.travis.yml

```diff
@@ -5,8 +5,11 @@ language: bash
 services:
   - docker
 
-install:
-  - docker-compose build
+before_install:
+  - export APP_IMAGE=keboola-db-extractor-logger
+  - docker -v
+  - docker build -t $APP_IMAGE .
+  - docker run $APP_IMAGE composer ci
 
 script:
-  - docker-compose run --rm app
+  - docker-compose run --rm dev
```

Comment:
tady imho měl ten app zůstat - abys testoval to co je v tom image a ne to co je na disku...

Reply:
ale tím, že je to knihovna a ne komponenta tak je to docela jedno

Reply:
je to zpět